ftplugin/supercollider.vim

```diff
@@ -209,18 +231,69 @@ endfunction
 
 let s:sclangStarted = 0
 
-function SClangStart()
-    if $TERM[0:5] == "screen"
-        if executable("tmux")
-            call system("tmux split-window -p 20 ; tmux send-keys " . s:sclangPipeApp . " Enter ; tmux select-pane -U")
-            let s:sclangStarted = 1
-        else
-            echo "Sorry, screen is not supported yet.."
-        endif
-    else
-        call system(s:sclangTerm . " " . s:sclangPipeApp . "&")
-        let s:sclangStarted = 1
-    endif
+function SClangStart(...)
+	if $TERM[0:5] == "screen"
```

Comment:
This should not use $TERM, as you can set different TERMs in tmux (although tmux-256color is standardized since 2.2, so this would not detect it correctly anyways).
A better way to detect whether you're in tmux or screen is to check whether the $TMUX or the $STY environment variable is set (respectively).

Reply:
@dvzrv AFAIU this is only used to check if the user is using a multiplexer only..
A couple of lines below you can see the other checks:

```vim
if $TERM[0:5] == "screen"
  " ...
  if executable("tmux")
    " ...
  if executable("screen")
    " ...
```

Let me know if that should be changed or improved!
ps: the original code is not from me, I only simplified it and cleaned it up